### ISSUE TITLE
ProjectReference -> Package Dependency: cannot convert from 'System.Security.Cryptography.ECCurve' to 'int' 

### ISSUE THREAD

**Issue by chcosta** [dotnet/corefx#8721]
A recent change to System.Security.Cryptography.OpenSsl changed the surface area of the library.

The result is that if you try to build tests against packages (convert projectrefrence’s into package dependencies), you get a build break.
```
2016-05-20T07:00:03.3568246Z EcDsaOpenSslProvider.cs(21,37): error CS1503: Argument 1: cannot convert from 'System.Security.Cryptography.ECCurve' to 'int' [D:\A\_work\1\s\src\System.Security.Cryptography.OpenSsl\tests\System.Security.Cryptography.OpenSsl.Tests.csproj]
2016-05-20T07:00:03.3568246Z EcDsaOpenSslTests.cs(250,35): error CS1503: Argument 1: cannot convert from 'System.Security.Cryptography.ECCurve' to 'int' [D:\A\_work\1\s\src\System.Security.Cryptography.OpenSsl\tests\System.Security.Cryptography.OpenSsl.Tests.csproj]
2016-05-20T07:00:03.3568246Z EcDsaOpenSslTests.cs(265,35): error CS1503: Argument 1: cannot convert from 'System.Security.Cryptography.ECCurve' to 'int' [D:\A\_work\1\s\src\System.Security.Cryptography.OpenSsl\tests\System.Security.Cryptography.OpenSsl.Tests.csproj]
2016-05-20T07:00:03.3568246Z EcDsaOpenSslTests.cs(274,35): error CS1503: Argument 1: cannot convert from 'System.Security.Cryptography.ECCurve' to 'int' [D:\A\_work\1\s\src\System.Security.Cryptography.OpenSsl\tests\System.Security.Cryptography.OpenSsl.Tests.csproj]
```

I am creating a PR to disable this library from the conversion until this issue has been resolved.  This issue is tracking the problem, and the temporary work-around.

/cc @steveharter , @bartonjs

**Comment by steveharter** [comment]
This may be fixed by #8709

**Comment by bartonjs** [comment]
#8732 is the updated version of trying to fix this
